config/cors.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+return [
+
+    /*
+     * A cors profile determines which origins, methods, headers are allowed for
+     * a given requests. The `DefaultProfile` reads its configuration from this
+     * config file.
+     *
+     * You can easily create your own cors profile.
+     * More info: https://github.com/spatie/laravel-cors/#creating-your-own-cors-profile
+     */
+    'cors_profile' => Spatie\Cors\CorsProfile\DefaultProfile::class,
+
+    /*
+     * This configuration is used by `DefaultProfile`.
+     */
+    'default_profile' => [
+
+        'allow_credentials' => false,
+
+        'allow_origins' => [
+            '*',
```

Comment:
Should be updated in production. Perhaps we should use an `env` variable? @Walkerguy